packages/studio-plugin/src/types/ComponentMetadata.ts

```diff
@@ -9,4 +9,5 @@ export type ComponentMetadata = {
   initialProps?: PropValues;
   propShape?: PropShape;
   acceptsChildren?: boolean;
+  cssImports?: string[];
```

Comment:
do we want this field to be required like it is in PageState? it looks like it'll always be specified, even if it's just an empty array